openhands/core/config/app_config.py

```diff
@@ -52,6 +52,7 @@ class AppConfig:
     sandbox: SandboxConfig = field(default_factory=SandboxConfig)
     security: SecurityConfig = field(default_factory=SecurityConfig)
     runtime: str = 'docker'
+    runtime_executor: str = 'openhands.runtime.executor:ActionExecutor'
```

Comment:
The colon will become dot if we go to using `get_impl` in `openhands.utils.import_utils`. 

So: `openhands.runtime.executor.ActionExecutor`